tests/unit/src/test/scala/tests/MillVersionSuite.scala

```diff
@@ -1,8 +1,10 @@
-package tests
+package scala.meta.internal.builds.tests
```

Comment:
Looks like this wasn't intentional actually?

Reply:
It was, for the sake of `getMillVersion` being package private.